src/app/modules/library/library-project-disciplines/library-project-disciplines.component.ts

```diff
@@ -1,20 +1,22 @@
 import { Component, Input, OnInit } from '@angular/core';
 import { LibraryProject } from '../libraryProject';
+import { CommonModule } from '@angular/common';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { MatTooltipModule } from '@angular/material/tooltip';
 
 @Component({
   selector: 'app-library-project-disciplines',
   templateUrl: './library-project-disciplines.component.html',
-  styleUrls: ['./library-project-disciplines.component.scss']
+  styleUrls: ['./library-project-disciplines.component.scss'],
```

Comment:
Use new styleUrl feature in Angular 17?
```suggestion
  styleUrl: './library-project-disciplines.component.scss',
```